collect_app/src/androidTest/java/org/odk/collect/android/support/pages/MainMenuPage.java

```diff
@@ -27,7 +27,7 @@ public class MainMenuPage extends Page<MainMenuPage> {
     @Override
     public MainMenuPage assertOnPage() {
         return WaitFor.waitFor(() -> {
-            onView(withText(R.string.enter_data)).check(matches(isDisplayed()));
+            onView(withText(R.string.enter_data_button)).check(matches(isDisplayed()));
```

Comment:
It was a bug.
`enter_data` is a title displayed when we open the list of forms
`enter_data_button` is the title of the button we use to open the list of forms.
In English, both hold the same string but it's not always the case in all other languages.